src/core/symbology/qgspointdisplacementrenderer.cpp

```diff
@@ -243,8 +256,9 @@ void QgsPointDisplacementRenderer::calculateSymbolAndLabelPositions( QgsSymbolRe
   }
   else if ( nPosition == 1 ) //If there is only one feature, draw it exactly at the center position
   {
+    double side = sqrt( pow( symbolDiagonal, 2 ) / 2.0 );
```

Comment:
Perhaps ```const``` ?

Reply:
Are you suggesting the side variable to be const or symbolDiagonal should be const?

side is only used once for each single symbol, but it can have different values for each single symbol. symbolDiagonal is the maximum diagonal for each group (and when it's a single symbol its equal to the single symbol diagonal). I guess the side could be const for a single symbol, but it's only used in one place for single symbol so I'm not sure what the benefit would be.

Reply:
just better practice i guess? so someone doesn't accidentally modify it